packages/material-ui/src/index.d.ts

```diff
@@ -7,7 +7,7 @@ export { StyledComponentProps };
  * certain `classes`, on which one can also set a top-level `className` and inline
  * `style`.
  */
-export type StandardProps<C, ClassKey extends string, Removals extends keyof C = never> = Omit<
```

Comment:
Why not just redefine `Omit` to allow using any valid key type?

```ts
export type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
```

Reply:
Yes. It is better. Thank you! 👍